Fix:
error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]
--- xmalloc.c.orig	2022-01-10 14:26:37.000000000 -0600
+++ xmalloc.c	2022-01-10 14:31:00.000000000 -0600
@@ -25,6 +25,7 @@
 # define VOID char
 #endif
 
+#include <string.h>
 #include <sys/types.h>
 
 #if STDC_HEADERS
